base/sparse/sparsematrix.jl

```diff
@@ -171,19 +171,17 @@ function Base.show(io::IOContext, S::SparseMatrixCSC)
         count == print_count && break
     end
 
-    lines_bottom = String[]
     if !will_fit
-        count = 0
-        for col = reverse(1:S.n), r = reverse(nzrange(S, col))
-            count += 1
-            push!(lines_bottom, _format_line(r, col))
-            count == print_count && break
+        print(io, "\n  \u22ee")
+        # find the column to start printing in for the last print_count elements
+        nextcol = searchsortedfirst(S.colptr, nnz(S) - print_count + 1)
+        for r = nnz(S) - print_count + 1 : S.colptr[nextcol] - 1
```

Comment:
Though not strictly necessary, parens would make the range much easier to parse?